src/reason_pprint_ast.ml

```diff
@@ -405,6 +405,9 @@ let rules = [
       (fun s -> (`Right, s = "asr"));
     ];
     [
+      (fun s -> (`Left, s.[0] == '#'));
```

Comment:
Just to confirm - this order in the rules matches the order of token precedence defined in `reason_parser.mly`?